### PR TITLE
Feat/remove code from ccg spec repo

### PR DESCRIPTION
**Pull request by OR13**
This work item is focused on the spec, I have removed the "experimental code" from the repo and linked to the appropriate github repos where that work can be contributed to.

This should make contributing to the spec the core mission of this repo.

### CODE REVIEW

**Review by mprorock — APPROVED** [review]

**Comment by mprorock** [comment]
@kimdhamilton think this is the right way to go - let's keep the CCG repo focused on the spec/standards work and have the code refs pointing out to OSS implementations as this PR covers

File: README.md [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+### Latest Specification
 
-### [Experimental Implementation](./packages/universal-wallet)
+- [https://w3id.org/wallet](https://w3id.org/wallet)
+
+### Latest Context
+
+- [https://w3id.org/wallet/v1](https://w3id.org/wallet/v1)
+
+### Experimental Implementations
```
Comment (OR13) [inline README.md:17]: See this section for links to OSS implementations of the spec

**Review by kimdhamilton — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
+## Universal Wallet Interoperability
 
 ![CI](https://github.com/w3c-ccg/universal-wallet-interop-spec/workflows/CI/badge.svg)
 
 A portable, extensible, JSON-LD wallet supporting digital currencies and credentials.
 
 ## Links
 
-### [Latest Specification](https://w3c-ccg.github.io/universal-wallet-interop-spec/)
+### Latest Specification
```
Comment (TallTed) [inline README.md:9]: Should the link to the spec not now be below the section header, instead of zero section content?

Comment (OR13) [reply README.md:9]: The link is preserved, see below (the bullet point that makes use of the perma-id redirect service).

**Pull request merged**